JS/fluentRevealNavbar.uc.js

```diff
@@ -16,14 +16,28 @@
       // if true, show the effect on bookmarks on the toolbar
       includeBookmarks: true,
 
+      // if true, show the effect on the urlbar
+      includeUrlBar: true,
+
       // the color of the gradient. default is sort of a faint baby blue. you may prefer just white, e.g. hsla(0, 0%, 100%, 0.05)
-      lightColor: "hsla(224, 100%, 80%, 0.15)",
+      // lightColor: "hsla(224, 100%, 80%, 0.15)",
```

Comment:
You can remove these lines

Reply:
I mean the old lines you commented out btw. Not sure why github is showing it like this